CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * Remove package emacs24-nox ,to work arround issue #114
 * Add Debian 10 support
 * Drop Ubuntu 14.04 support
+* Drop Ansible 2.4.0 and 2.5.0 support
```

Comment:
@nickovivar `v1.6.2` is already a released version, I think you need to bump the version here.